functional_tests/scylla_operator/test_functional.py

```diff
@@ -332,3 +332,21 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
 
     not_running_pods = ','.join([pods_info[0] for pods_info in scylla_operator_pods if pods_info[1] != 'Running'])
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
+
+
+def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl(f'get pods -A -l "scylla/cluster=sct-cluster" -o yaml')
```

Comment:
Do not hardcode "sct-cluster". It is value from our "k8s_scylla_cluster_name" config option.
So, either use config option here or just get pods by "app.kubernetes.io/name=scylla" label.
I prefer the latter one.

Reply:
changed to "app.kubernetes.io/name=scylla"